utils/graphqlHepers.ts

```diff
@@ -0,0 +1,227 @@
+import { gql } from 'graphql-request'
+import { gqlClient, graphQLClientRequestWithRetry } from './graphql'
+import type { HealthcareProfessional,
+    HealthcareProfessionalSearchFilters,
+    Facility,
+    FacilitySearchFilters,
+    Submission,
+    SubmissionSearchFilters } from '~/typedefs/gqlTypes'
+
+/**
```

Comment:
I would put this back into the stores as they're not too long yet. In most cases, I would say move it out around 1-2k lines OR if the work is not directly related to the store logic.